addons/sys_bf888s/script_component.hpp

```diff
@@ -0,0 +1,26 @@
+#define COMPONENT sys_bf888s
+#define COMPONENT_BEAUTIFIED BF 888S
+#include "\idi\acre\addons\main\script_mod.hpp"
+
+//#define DEBUG_MODE_FULL
+//#define DISABLE_COMPILE_CACHE
+//#define ENABLE_PERFORMANCE_COUNTERS
+
+#ifdef DEBUG_ENABLED_SYS_BF888S
+    #define DEBUG_MODE_FULL
+#endif
+
+#ifdef DEBUG_ENABLED_SYS_BF888S
+    #define DEBUG_SETTINGS DEBUG_SETTINGS_SYS_BF888S
+#endif
+
+#include "\idi\acre\addons\main\script_macros.hpp"
+
+#define NAME_BF888S Beofeng 888S
+
+#define GET_VAR(var1) acre_player getVariable QGVAR(var1)
```

Comment:
I would avoid using these defines. They might be confusing.

Reply:
I'm not sure what the alternatives would be? I started with (evidently) the 343 and it came from there.